pkg/crc/cluster/cert-renewal.go

```diff
@@ -28,7 +28,7 @@ func waitForPendingCsrs(ocConfig oc.Config) error {
 		return nil
 	}
 
-	return errors.RetryAfter(120, waitForPendingCsr, time.Second*5)
+	return errors.RetryAfter(120*time.Second, waitForPendingCsr, time.Second*5)
```

Comment:
@guillaumerose before this change this check used to run around 120*(waitforpendingProcess + 5) seconds but now it will be just 120 seconds and on slow system I am sure that is not going to enough and might fail so we should atleast consider to have it 5-8 mins if not 10-12 mins.

Reply:
If there was a problem and the CSR never comes, we cannot really expect the user to wait for 10-12 minutes before reporting the error.

Reply:
It's good to have this debate! I changed to 8min.